lib/tulul_stats_bot/tulul_stats_bot.rb

```diff
@@ -11,7 +11,11 @@ def self.start
 
           query = /\/top_(.+)/.match(message.text).captures[0] rescue nil
           query = query.split('@')[0] rescue nil
-          if query && (TululStats::User.fields.keys.reject{ |field| TululStats::User::EXCEPTION.include?(field) } + TululStats::Entity::ENTITY_QUERY + TululStats::IsTime::TIME_QUERY).include?(query)
+          if /^\/last_tulul([@].+)?/.match(message.text.strip)
```

Comment:
Any suggestion? All top_X is predefined in the `Entity` and `IsTime`. Meanwhile, this is not a _top_ query, this is "state" query. I was considering to make a [circular buffer](https://en.wikipedia.org/wiki/Circular_buffer) for this by using Mongodb's array push and pop, and make calculation as a sliding window and tell the _activity level_ of the user so it is equal to top_X query, but it is too much work for now.